Mail/Views/Thread/Message/InlineAttachmentWorker.swift

```diff
@@ -34,7 +34,7 @@ final class InlineAttachmentWorker: ObservableObject {
     @Published var presentableBody: PresentableBody
 
     /// Set to true when done processing
-    @Published var isMessagePreprocessed: Bool
+    private var isMessagePreprocessed: Bool
```

Comment:
We do not need to redraw once the `isMessagePreprocessed` is set to TRUE, the correct body is been processed.